registrations/migrations/0023_signup_and_signup_group_user_data.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 3.2.20 on 2023-09-12 13:03
+
+from django.db import migrations, models
```

Comment:
I think at least the signup model part of this should be in the first  [commit](https://github.com/City-of-Helsinki/linkedevents/pull/708/commits/c839fb6809c823ce87220197b452880cfaa7fa9e) since this overrides many things introduced there